javaagent/javaagent.gradle

```
@@ -123,3 +123,5 @@ tasks.withType(Test).configureEach {
 
   dependsOn shadowJar
 }
+
+assemble.dependsOn shadowJar
```

I think good to build both on assemble

interestingly, this is exactly what didn't work for me in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/708#issuecomment-674670104, but just tried and looks like working now, so yay?!

```suggestion
assemble.dependsOn lightShadow
assemble.dependsOn shadowJar
```